app/customer.py

```diff
@@ -0,0 +1,74 @@
+from math import dist
+from typing import Dict, Any
+
+from app.car import Car
+
+
+class Customer:
+    def __init__(self, customer_data: Dict[str, Any]) -> None:
```

Comment:
When you create Customer object, you should every time look here through all the attributes to be sure that u put right customer_data dict here. Don't do that, make it easier